packages/connect-finance/src/entities/Finance.ts

```diff
@@ -0,0 +1,38 @@
+import { Address, SubscriptionHandler } from '@aragon/connect-types'
+import Transaction from './Transaction'
+import TokenBalance from './TokenBalance'
+import { IFinanceConnector } from '../types'
+
+export default class Finance {
+  #appAddress: Address
+  #connector: IFinanceConnector
+
+  constructor(connector: IFinanceConnector, appAddress: Address) {
+    this.#appAddress = appAddress
+    this.#connector = connector
+  }
+
+  async disconnect() {
+    await this.#connector.disconnect()
+  }
+
+  async transactions({ first = 1000, skip = 0 } = {}): Promise<Transaction[]> {
+    return this.#connector.transactionsForApp(this.#appAddress, first, skip)
+  }
+
+  onTransactions(callback: Function): SubscriptionHandler {
```

Comment:
I updated the voting connector recently, to align the parameters of `votes` with `onVotes` (https://github.com/aragon/connect/pull/135/commits/a2dbb76c1c5b132d1c46cb793bd1acef19007408). I think we could do the same here!